### PR TITLE
Add/export

### PR DESCRIPTION
**Pull request by aitordiaz**

### CODE REVIEW

File: src/averell/cli.py [hunk 1]
```diff
@@ -13,18 +14,28 @@ def main():
 
 
 @main.command()
-@click.option('--output', default=Path.cwd() / "corpora", help='Folder to download')
-@click.option('--granularity', default=None, help='Granularity')
-@click.argument('ids', nargs=-1)
-def download(ids, granularity, output):
+@click.option('--output', default=Path.cwd() / "corpora",
+              help='Folder to download')
+@click.argument('ids', nargs=-1, type=click.INT)
```
Comment (versae) [inline src/averell/cli.py:19]: :100:

File: src/averell/core.py [hunk 2]
```diff
@@ -32,36 +31,68 @@ def get_corpora(corpus_indices=None, granularity=None,
...
+                    else:
+                        if not (Path(corpora_folder) / corpus_folder).exists():
+                            logging.error(
+                                f'{CORPORA_SOURCES[corpus_id]["name"]}'
+                                f' not downloaded')
+                            continue
+                        granularities_list = CORPORA_SOURCES[corpus_id][
+                            "properties"]["granularity"]
+                        if granularity not in granularities_list:
+                            corpus_name = CORPORA_SOURCES[corpus_id]["name"]
```
Comment (versae) [inline src/averell/core.py:80]: Would this chunk become cleaner if we assigned `corpus = CORPORA_SOURCES[corpus_id]` at the beginning? Just wondering.

File: src/averell/utils.py [hunk 3]
```diff
@@ -247,3 +268,21 @@ def write_json(poem_dict, filename):
...
+def read_features(corpus_folder):
+    """
+    Read the dictionary of each poem in "corpus_folder" and
+    return the list of python dictionaries
+    :param corpus_folder: Local folder where the corpus is located
+    :return: List of python dictionaries with the poems features
+    """
+    features_path = Path.cwd() / Path(corpus_folder) / "averell" / "parser"
+    # features = [json.loads(json_file.read_text()) for json_file in
+    #            features_path.rglob("*.json")]
```
Comment (versae) [inline src/averell/utils.py:282]: Remove

File: src/averell/core.py (hunk 2 above)
Comment (aitordiaz) [reply src/averell/core.py:80]: Yes, like `corpus_folder = CORPORA_SOURCES[corpus_id]["properties"]["folder_name"]`, and assign all the variables at the beginning?

File: src/averell/utils.py (hunk 3 above)
Comment (aitordiaz) [reply src/averell/utils.py:282]: Done

**Pull request merged**